sorc/chgres_cube.fd/input_data.F90

```diff
@@ -49,56 +49,56 @@ module input_data
 
 ! Fields associated with the atmospheric model.
 
- type(esmf_field), public              :: dzdt_input_grid       ! vert velocity
- type(esmf_field)                      :: dpres_input_grid      ! pressure thickness
- type(esmf_field), public              :: pres_input_grid       ! 3-d pressure
- type(esmf_field), public              :: ps_input_grid         ! surface pressure
- type(esmf_field), public              :: terrain_input_grid    ! terrain height
- type(esmf_field), public              :: temp_input_grid       ! temperature
- type(esmf_field)                      :: u_input_grid          ! u/v wind at grid
- type(esmf_field)                      :: v_input_grid          ! box center
- type(esmf_field), public              :: wind_input_grid       ! 3-component wind
- type(esmf_field), allocatable, public :: tracers_input_grid(:) ! tracers
-
- integer, public                 :: lev_input      ! # of atmospheric layers
- integer, public                 :: levp1_input    ! # of atmos layer interfaces
+ type(esmf_field), public              :: dzdt_input_grid       !< vert velocity
+ type(esmf_field)                      :: dpres_input_grid      !< pressure thickness
+ type(esmf_field), public              :: pres_input_grid       !< 3-d pressure
+ type(esmf_field), public              :: ps_input_grid         !< surface pressure
+ type(esmf_field), public              :: terrain_input_grid    !< terrain height
+ type(esmf_field), public              :: temp_input_grid       !< temperature
+ type(esmf_field)                      :: u_input_grid          !< u/v wind at grid
+ type(esmf_field)                      :: v_input_grid          !< box center
+ type(esmf_field), public              :: wind_input_grid       !< 3-component wind
+ type(esmf_field), allocatable, public :: tracers_input_grid(:) !< tracers
+
```

Comment:
The pound sign gives a very large font in doxygen.  Can you try replacing "#" with "number"